scripts/emissions/quarterly_emissions.py

```diff
@@ -0,0 +1,134 @@
+import os
+from decimal import Decimal
+import json
+from datetime import date, datetime, timedelta
+from rich.console import Console
+
+from brownie import Wei, network
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+C = Console()
+
+# 7 days
+WEEK_DURATION = 604800
+
+
+def main(target_file="generated_emissions_info_BIP_88_Emissions"):
+    path = os.getcwd() + f"/scripts/emissions/{target_file}.json"
+    with open(path) as f:
+        data = json.load(f)
+
+    # common times for Arb & Eth chain
+    today = date.today()
+    target_week = find_thrusday(today)
+
+    emissions_data = data[target_week.strftime("%d-%m-%y")]
+    time_range = emissions_data["timerange"]
+
+    last_quarter_time_rage = list(data.keys())[-1]
+    end_quarter_timestamp = data[last_quarter_time_rage]["timerange"]["endtime"]
+
+    duration = end_quarter_timestamp - time_range["starttime"]
+
+    weeks_no = duration / WEEK_DURATION
```

Comment:
```suggestion
    weeks_no = duration / WEEK_DURATION
    assert weeks_no == duration // WEEK_DURATION
```
i think we need a round number here, which it looks like works out in this case, but this could be enforced with `weeks_no = duration // WEEK_DURATION`.

or maybe even better would be adding an extra `assert weeks_no == duration // WEEK_DURATION` to catch possible mistakes in the `duration`.

Reply:
sounds cool enforcing more defensive coding here 👍